database/mssql/scripts/dbscripts/PSP_PIMS_S31_00/Alter Up/01_PSP_PIMS_S30_00_Alter_Up.sql

```diff
@@ -0,0 +1,1087 @@
+	[DISPLAY_ORDER]             	int NULL,
+	[CONCURRENCY_CONTROL_NUMBER]	bigint NOT NULL CONSTRAINT [DOCSTY_CONCURRENCY_CONTROL_NUMBER_DEF]  DEFAULT ((1)),
+	[DB_CREATE_TIMESTAMP]       	datetime NOT NULL CONSTRAINT [DOCSTY_DB_CREATE_TIMESTAMP_DEF]  DEFAULT (getutcdate()),
+	[DB_CREATE_USERID]          	nvarchar(30) NOT NULL CONSTRAINT [DOCSTY_DB_CREATE_USERID_DEF]  DEFAULT (user_name()),
+	[DB_LAST_UPDATE_TIMESTAMP]  	datetime NOT NULL CONSTRAINT [DOCSTY_DB_LAST_UPDATE_TIMESTAMP_DEF]  DEFAULT (getutcdate()),
+	[DB_LAST_UPDATE_USERID]     	nvarchar(30) NOT NULL CONSTRAINT [DOCSTY_DB_LAST_UPDATE_USERID_DEF]  DEFAULT (user_name()),
+	CONSTRAINT [DOCSTY_PK] PRIMARY KEY CLUSTERED([DOCUMENT_STATUS_TYPE_CODE])
+ ON [PRIMARY])
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Code value of the document status type.' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_DOCUMENT_STATUS_TYPE', 
+	@level2type = N'Column', @level2name = N'DOCUMENT_STATUS_TYPE_CODE'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Descriptive translation of the document status type code.' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_DOCUMENT_STATUS_TYPE', 
+	@level2type = N'Column', @level2name = N'DESCRIPTION'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Indicates if the code is still in use.' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_DOCUMENT_STATUS_TYPE', 
+	@level2type = N'Column', @level2name = N'IS_DISABLED'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Determines the default display order of the code descriptions.' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_DOCUMENT_STATUS_TYPE', 
+	@level2type = N'Column', @level2name = N'DISPLAY_ORDER'
+GO
+EXEC sp_addextendedproperty 
+	@name = N'MS_Description', @value = N'Table describing the available document types.' , 
+	@level0type = N'Schema', @level0name = N'dbo', 
+	@level1type = N'Table', @level1name = N'PIMS_DOCUMENT_STATUS_TYPE'
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create sequence dbo.PIMS_DOCUMENT_TYPE_ID_SEQ
+PRINT N'Create sequence dbo.PIMS_DOCUMENT_TYPE_ID_SEQ'
+GO
+CREATE SEQUENCE [dbo].[PIMS_DOCUMENT_TYPE_ID_SEQ]
+	AS bigint
+	START WITH 1
+	INCREMENT BY 1
+	MINVALUE 1
+	MAXVALUE 2147483647
+	NO CYCLE
+	CACHE 50
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create table dbo.PIMS_DOCUMENT_TYPE
+PRINT N'Create table dbo.PIMS_DOCUMENT_TYPE'
+GO
+CREATE TABLE [dbo].[PIMS_DOCUMENT_TYPE]  ( 
+	[DOCUMENT_TYPE_ID]              	bigint NOT NULL CONSTRAINT [DOCTYP_DOCUMENT_TYPE_ID_DEF]  DEFAULT (NEXT VALUE FOR [PIMS_DOCUMENT_TYPE_ID_SEQ]),
+	[MAYAN_ID]                      	bigint NOT NULL CONSTRAINT [DOCTYP_MAYAN_ID_DEF]  DEFAULT ('<Empty>'),
```

Comment:
pardon my SQL ignorance but can a string - ie **'<Empty>'** be a default value on a BIGINT field?

Reply:
Weird.